frontend/src/Components/Results/Results.tsx

```diff
@@ -46,6 +47,52 @@ interface ResultDisplayProps {
 }
 function ResultDisplay(props: ResultDisplayProps) {
   const { result } = props;
+
+// CUMULATIVE DEATHS PER AGENT CODE
+// Finding which agent types are duplicated and their respective indices
+// Adding the cumulative deaths, and removing duplicated agentTypes
+  var agentTypes = result.deaths.map((d) => d.agentType) 
+  var cumulativeDeaths = result.deaths.map((d) => d.cumulativeDeaths) 
+  var myMap = new Map<string, number[]>()
+
+
+  for (var i = 0; i < agentTypes.length; i++) {
+      var element = agentTypes[i].toString();  
+      var test = myMap.get(element)
+      if (typeof test != "undefined"){
+        test.push(i)
+        myMap.set(element,test)
+      }else {
+        myMap.set(element,[i])
+      }
+  }
+
+  var findDuplicates = Array.from(myMap.entries()).filter(
+    ([key, val]) =>{
+      if (val.length > 1){
+        return [key, val]
+      }
+    });
+    
+  if (findDuplicates){
+    for (var j=0; j<findDuplicates.length; j++){
+      let [n, idx] = findDuplicates[j]
+      var tempValue = 0
+      for(var i = 0; i<idx.length; i++){
+        tempValue += cumulativeDeaths[idx[i]]
+      }
+      cumulativeDeaths[idx[0]] = tempValue
+
+      for (var z = 1; z<idx.length; z++){
+        delete cumulativeDeaths[idx[z]]
+        delete agentTypes[idx[z]]
+      }
+    }
+    cumulativeDeaths = cumulativeDeaths.filter(function(e){return e});
+    agentTypes = agentTypes.filter(function(e){return e});
+
+  }
```

Comment:
Can be moved to `frontend/src/Helpers/Utils.ts` into a function with input `Death[]` and output two lists

Reply:
Moved to utils in the most recent commit